content/docs/concepts/peripherals/eeprom.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "EEPROM"
+weight: 3
+description: |
+  How to use the internal EEPROM.
+---
+
+For API documentation, see the [machine API](../../../reference/machine#eeprom).
+
+The EEPROM is a form of non-volatile memory that is uusually integrated along side an MCU. EEPROMs allows you to store a relatively small amount of data by allowing bytes to be erased and reprogrammed. For the most part, EEPROMs are not integrated directly onto a board, but in some cases they are provided as a standard peripheral.
+
+Although it may seem useful to have some type of data storage for your applications, the EEPROM is often limited in the amount of write cycles that it contains. This means that after a certain amount of writes, the EEPROM will not writeable, but you may still read data from it.
```

Comment:
EEPROMs are almost always included directly on the chip, I think?
Also, here is a suggestion for the text:

```suggestion
An EEPROM is a form of non-volatile memory that is usually integrated in an MCU. EEPROMs allow you to store relatively small amounts of data by allowing bytes to be erased and reprogrammed.

EEPROMs, like flash, can only be written a limited number of times but are not limited in how often they can be read. Therefore, they are mainly useful to store configuration values that do not change very often. The difference with flash is that individual bytes can be erased and that they usually have a longer lifespan.
```